lib/compress/zstd_compress.c

```diff
@@ -1512,6 +1529,11 @@ static size_t ZSTD_estimateCCtxSize_usingCCtxParams_internal(
 
     size_t const cctxSpace = isStatic ? ZSTD_cwksp_alloc_size(sizeof(ZSTD_CCtx)) : 0;
 
+    size_t const maxNbExternalSeq = ZSTD_sequenceBound(ZSTD_BLOCKSIZE_MAX);
```

Comment:
Same comment here, regarding `blockSize <= ZSTD_BLOCKSIZE_MAX`, leading to a leaner memory budget.